picard/ui/theme.py

```diff
@@ -33,6 +39,74 @@
 )
 
 
+OS_SUPPORTS_THEMES = True
+if IS_MACOS:
+    def is_dark_theme_supported():
+        import platform
+        try:
+            current_version = tuple(map(int, platform.mac_ver()[0].split(".")))[:2]
+        except ValueError:
+            log.warning("Error while converting the MacOS version string into a tuple: %s" % platform.mac_ver()[0])
+            return False
+
+        mojave_version = (10, 14)  # Dark theme support was introduced in Mojave
+        return current_version >= mojave_version
+    OS_SUPPORTS_THEMES = is_dark_theme_supported()
+    del is_dark_theme_supported
+
+elif IS_HAIKU:
+    OS_SUPPORTS_THEMES = False
+
+
+@unique
+# Those are values stored in config file:
+class UiTheme(IntEnum):
+    DEFAULT = 1
+    DARK = 2
+    LIGHT = 3
+    SYSTEM = 4
+
```

Comment:
Since we now use names in config file, we don't have much need of `IntEnum`, using something like:

```python
class UiTheme(Enum):

    DEFAULT = 'default'
    DARK = 'dark'
    LIGHT = 'light'
    SYSTEM = 'system'

    def __str__(self):
        return self.value
```

would allow us to get rid of int/name conversion.

`str(UiTheme.DARK)` being `dark`

Reply:
We've gone full circle. ._.

Reply:
Hmm, here was I thinking this would work. =x
```
_UI_THEME_LABELS = {}
_UI_THEME_LABELS.update([(theme_entry, N_(theme_entry.value.capitalize())) for theme_entry in list(UiTheme)])
```
Capitalizing the theme in UiTheme and moving the N_ stuff there would save a few more lines.

Reply:
Nope, you can't do that.

`N_()` is a noop function to mark strings for translation. It should never (ever) be called with anything but a string.